generators/server/templates/src/main/java/package/config/_JHipsterProperties.java

```diff
@@ -222,7 +222,7 @@ public void setFrom(String from) {
 
         private final RememberMe rememberMe = new RememberMe();
         <%_ } _%>
-        <%_ if (authenticationType == 'oauth2' || authenticationType == 'jwt') { _%>
+        <%_ if (authenticationType == 'oauth2' || authenticationType == 'jwt' || applicationType == 'uaa' || applicationType == 'gateway') { _%>
```

Comment:
here as well `authenticationType == 'jwt'` should suffice for gateway appType unless you are saying that you wont set `authenticationType = 'jwt'` for gateway